src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1320,7 +1320,7 @@ private string GetGetTypeInfoImplementation()
 
                 sb.Append(
 @$"/// <inheritdoc/>
-public override {JsonTypeInfoTypeRef} GetTypeInfo({TypeTypeRef} type)
+public override {JsonTypeInfoTypeRef}? GetTypeInfo({TypeTypeRef} type)
```

Comment:
I assume the base method is annotated to return a nullable?

Reply:
That's right.